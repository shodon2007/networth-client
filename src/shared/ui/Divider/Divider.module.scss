.divider {
	height: 2px;
	width: 100%;
	background-color: rgba(255, 255, 255, 0.3);
	border-radius: 10px;
	margin: 10px 0;
}
