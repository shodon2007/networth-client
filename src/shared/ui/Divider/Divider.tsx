import cls from "./Divider.module.scss";

const Divider = () => {
	return <div className={cls.divider} />;
};

export default Divider;
