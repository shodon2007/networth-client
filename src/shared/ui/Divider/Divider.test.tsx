import {render} from "@testing-library/react";
import Divider from "./Divider";

describe("testing Divider.tsx", () => {
	it("should pass snapshot testing", () => {
		const screen = render(<Divider />);
		const divider = screen.getByTestId("divider");
		expect(divider).toMatchSnapshot();
	});
	it("should be in the document", () => {
		const screen = render(<Divider />);
		const divider = screen.getByTestId("divider");
		expect(divider).toBeInTheDocument();
	});
});
