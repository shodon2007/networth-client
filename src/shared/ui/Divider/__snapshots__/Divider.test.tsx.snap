// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Divider.tsx should pass snapshot testing 1`] = `
<div
  class="divider"
  data-testid="divider"
/>
`;
